context/PoolContext/helpers.ts

```diff
@@ -24,68 +23,49 @@ import { ethers, BigNumber as EthersBigNumber } from 'ethers';
  * @returns a Pool object
  */
 export const initPool: (
-    pool: PoolType,
+    pool: StaticPoolInfo,
     provider: ethers.providers.JsonRpcProvider | ethers.providers.WebSocketProvider,
 ) => Promise<Pool> = async (pool, provider) => {
     const contract = new ethers.Contract(pool.address, LeveragedPool__factory.abi, provider) as LeveragedPool;
 
-    const [
-        updateInterval,
-        lastUpdate,
-        shortBalance,
-        longBalance,
-        oraclePrice,
-        quoteToken,
-        longToken,
-        shortToken,
-        poolCommitter,
-        leverageAmount,
-    ] = await Promise.all([
-        contract.updateInterval(),
-        contract.lastPriceTimestamp(),
-        contract.shortBalance(),
-        contract.longBalance(),
-        contract.getOraclePrice(),
-        contract.quoteToken(),
-        contract.tokens(0),
-        contract.tokens(1),
-        contract.poolCommitter(),
-        contract.leverageAmount(),
-    ]);
-
-    const [frontRunningInterval, keeper, name] = await Promise.all([
-        contract.frontRunningInterval(),
-        contract.keeper(),
-        contract.poolName(),
+    const [lastUpdate, shortBalance, longBalance, oraclePrice, poolCommitter] = await Promise.all([
+        contract.lastPriceTimestamp({
+            blockTag: 'latest',
+        }),
+        contract.shortBalance({
+            blockTag: 'latest',
+        }),
+        contract.longBalance({
+            blockTag: 'latest',
+        }),
+        contract.getOraclePrice({
+            blockTag: 'latest',
+        }),
+        contract.poolCommitter({
+            blockTag: 'latest',
+        }),
```

Comment:
I think `poolCommitter` just returns an address, should we add it to the static config?

Reply:
hmmmm, just realised that keeper, and committer are technically changeable although unlikely. Am just going to leave it as is but probably should keep it as fetching anything that can change